### PR TITLE
Remove @fields.message, clean up, add a test

### PR DESCRIPTION
**Pull request by jlee-made**

### CODE REVIEW

File: test_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-e git+https://github.com/jlee-made/icdiff.git@13abb1ae6120f31edcf856c7468268f1eeff6cea#egg=icdiff-inprocess
```
Comment (LuRsT) [inline test_requirements.txt]: Why do we need to use this specific commit?

File: tests/test_normal_case.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        censor('@fields.process', int)
+        censor('@fields.relativeCreated', float)
+        censor('@fields.thread', int)
+        censor('@timestamp', str)
+
+    def test(self):
+        self.log.debug('spam')
+        self.assertEqualDiff(
+            json.loads(self.stream.getvalue()),
+            {'@fields': {'args': [],
```
Comment (LuRsT) [inline tests/test_normal_case.py]: Odd indentation here and bellow at line 103 too.

File: test_requirements.txt (hunk 1 above)
Comment (jlee-made) [inline test_requirements.txt]: Why not?  It's a tremendous commit

File: tests/test_normal_case.py (hunk 2 above)
Comment (jlee-made) [inline tests/test_normal_case.py]: It's a paste.  I got tired of reformatting pastes.

File: test_requirements.txt (hunk 1 above)
Comment (egidijus) [inline test_requirements.txt]: Great response John Lee, but citation is needed..

**Pull request merged**